sdk/cosmos/azure-spring-data-cosmos/src/samples/java/com/azure/spring/data/cosmos/CosmosProperties.java

```diff
@@ -22,6 +22,8 @@ public class CosmosProperties {
 
     private boolean queryMetricsEnabled;
 
+    private int maxDegreeOfParallelism;
```

Comment:
I believe this file is used in the readme as a code snippet, as you can see in the file description at the top. Whenever you change these files, you will also need to update the readme with the correct line numbers and content of these code snippet files. 
Let's discuss offline and I can help you with the change.

Reply:
I did update the ReadMe, did you see something specific I missed?